### PR TITLE
ci: create a documentation issue template

### PR DESCRIPTION
**Pull request by slumbering**
Signed-off-by: jffarge <jf@dagger.io>

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/documentation.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        - anything else that doesn't require touching the codebase itself.
+
+        If you followed the documentation but things don't work, take some time to consider if it's the documentation or the code that's wrong. In the latter, prefer using the "bug" template.
+
+  - type: checkboxes
+    attributes:
+      label: Have you read the Contributing Guidelines on issues?
+      options:
+        - label: I have read the [Contributing Guidelines](https://github.com/dagger/.github/blob/main/CONTRIBUTING.md).
+          required: true
```
Comment (gerhard) [inline .github/ISSUE_TEMPLATE/documentation.yaml:22]: Just before users finish submitting this, I would mention this link with the goal of encouraging them to create a pull request. I would remove this checkbox and transform it into what is known in design psychology as a [Nudge](https://growth.design/psychology#nudge).

This is the text that I would add at the very bottom of the template instead:

```yaml
  - type: markdown
    attributes:
      value: |
        We appreciate you taking the time to suggest improvements to our documentation.
        If you want to become a Dagger contributor and help us improve this, we will appreciate it even more.

        [You can become a contributor in less than 10 minutes](https://github.com/dagger/.github/blob/main/CONTRIBUTING.md).
        If you try it out and it doesn't live up to this promise, we definitely want to know what would have made it better for you.
        All Dagger contributors will be forever grateful 🙌🏻
```

As a follow-up to this PR, I would do the same for the [bug report template](https://github.com/dagger/.github/blob/main/.github/ISSUE_TEMPLATE/bug_report.yaml) 😉. That was the same Nudge in action. How did it feel? 🙂

Comment (slumbering) [reply .github/ISSUE_TEMPLATE/documentation.yaml:22]: It sounds like a very good idea :).

**Review by gerhard — APPROVED** [review]
⛴

**Comment by gerhard** [comment]
End-result:
image

I will improve the length of the last two sentences as a follow-up PR to this.

**Pull request merged**